packages/editor/src/components/parts/web-service-process/WebServiceProcessPart.tsx

```diff
@@ -0,0 +1,47 @@
+import { IVY_SCRIPT_TYPES, WebServiceProcessData } from '@axonivy/inscription-protocol';
+import { PartProps, usePartDirty, usePartState } from '../../../components/editors';
+import { useValidations } from '../../../context';
+import { Input, Radio, useFieldset } from '../../../components/widgets';
+import { useWebServiceProcessData } from './useWebServiceProcessData';
+import { PathFieldset } from '../common';
+
+export function useWebServiceProcessPart(): PartProps {
+  const { config, defaultConfig, initConfig, reset } = useWebServiceProcessData();
+  const compareData = (data: WebServiceProcessData) => [data];
+  const validation = [...useValidations(['wsAuth']), ...useValidations(['wsTypeName'])];
+  const state = usePartState(compareData(defaultConfig), compareData(config), validation);
+  const dirty = usePartDirty(compareData(initConfig), compareData(config));
+  return { name: 'Web Service Process', state, reset: { dirty, action: () => reset() }, content: <WebServiceProcessPart /> };
+}
+
+const WebServiceProcessPart = () => {
+  const { config, update } = useWebServiceProcessData();
+
+  const qualifiedNameField = useFieldset();
+  const authField = useFieldset();
+
+  return (
+    <>
+      <PathFieldset label='Qualified name' {...qualifiedNameField.labelProps} path='wsTypeName'>
+        <Input
+          value={config.wsTypeName}
+          onChange={change => update('wsTypeName', change)}
+          type={IVY_SCRIPT_TYPES.STRING}
+          {...qualifiedNameField.inputProps}
+        />
+      </PathFieldset>
+      <PathFieldset label='Authentication' {...authField.labelProps} path='wsAuth'>
+        <Radio
+          value={config.wsAuth}
+          onChange={change => update('wsAuth', change)}
+          items={[
+            { label: 'None/Container', value: 'NONE' },
+            { label: 'WS Security', value: 'WS_SECURITY' },
+            { label: 'HTTP Basic', value: 'HTTP_BASIC' }
+          ]}

```

Comment:
this values here are known via the protocol. So I would suggest that we should at least add some type safety to them should they ever change.

Put into the workflow-data.ts file in the protocol/src/data an entry: 
```ts
export const WS_AUTH_TYPE = {
  NONE: 'None/Container',
  WS_SECURITY: 'WS Security',
  HTTP_BASIC: 'HTTP Basic'
} as const satisfies Record<WsAuth, string>;
```

then you can reuse them here by e.g.:
```ts
Object.entries(WS_AUTH_TYPE).map(([value, label]) => ({ label, value }))
```

Reply:
i had to implement it like this because otherwise "change" was not happy with the type. i always had the following message: Argument of type 'string' is not assignable to parameter of type 'WsAuth'.

onChange={(change) => update('wsAuth', change)}
items={Object.entries(WS_AUTH_TYPE).map(([value, label]) => ({
   label,
   value: value as keyof typeof WS_AUTH_TYPE
}))}

Reply:
or simply do: `onChange={change => update('wsAuth', change as string)}`